src/signin/input/Input.module.scss

```
@@ -0,0 +1,52 @@
+@import "@/styles/global.scss";
+
+.input {
+  display: flex;
+  width: 40rem;
+  padding: 1.8rem 1.5rem;
+  justify-content: center;
+  align-items: center;
+
+  border-radius: 0.8rem;
+  border: 0.1rem solid $color-gray20;
+  background-color: $color-white;
+}
+
+.input::placeholder {
+  color: $color-gray60;
+}
+
+.input:focus {
+  border: 0.1rem solid $color-primary;
+  background-color: $color-white;
+}
```

부모 선택자`(&`)를 활용해 주시면 더 좋을 것 같네요!
그리고 중복되는 부분은 작성하지 않아도 될 것 같고, border는 색상만 변경되는 만큼 명확히 border-color로만 작성해주시는 게 좋습니다.
```suggestion
.input {
  display: flex;
  width: 40rem;
  padding: 1.8rem 1.5rem;
  justify-content: center;
  align-items: center;

  border-radius: 0.8rem;
  border: 0.1rem solid $color-gray20;
  background-color: $color-white;
  
  &::placeholder {
    color: $color-gray60;
  }
  
  &:focus {
    border-color: $color-primary;
  }
}
```